concent_api/verifier/utils.py

```diff
@@ -99,11 +110,18 @@ def delete_file(file_path):
         logger.warning(f'File with path {file_path} was not deleted, exception: {exception}')
 
 
-def generate_blender_output_file_name(scene_file):
-    return f'{settings.VERIFIER_STORAGE_PATH}/out_{scene_file}'
+def generate_full_blender_output_file_name(scene_file, frame_number, output_format):
+    base_blender_output_file_name = generate_base_blender_output_file_name(scene_file)
+    if output_format == 'jpeg':
```

Comment:
This `output_format` went out.

Reply:
what do you mean? aren't we supposed to pass this param?

Reply:
No no, what I have meant is that `jpeg` is no longer value in `OutputFormat` enum, and we validate it at the beginning of the task, so I think this code will never be reached.

Reply:
ok, I get it. done (good catch)

Reply:
Also, `'jpeg'` should be in some constant or enum, not hard-coded here.

We should have mapping between format names and extensions.

Reply:
> ok, I get it. done (good catch)

Lets say that I am familiar with how this one works ;)

> We should have mapping between format names and extensions.

Right, but currently it is 1 to 1 I think.

Reply:
No, it isn't. There's `jpg` and `jpeg`. Blender expects one on the command line but uses the other as file extension. That's why @dybi added this `if` here.